src/hb-aat-layout-morx-table.hh

```diff
@@ -69,12 +73,12 @@ struct RearrangementSubtable
 	ret (false),
 	start (0), end (0) {}
 
-    inline bool is_actionable (StateTableDriver<EntryData> *driver HB_UNUSED,
+    inline bool is_actionable (StateTableDriver<MorxTypes, EntryData> *driver HB_UNUSED,
```

Comment:
Why do these say MorxTypes instead of Types?

Reply:
Ops, replace error